app/api/tutorials_api.rb

```diff
@@ -93,5 +99,77 @@ class TutorialsApi < Grape::API
       tutorial.destroy!
       tutorial
     end
+
+
+    ## ---- New Endpoints ----
+    # New tutorial endpoints to incorporate unit activity sets
+    # These will eventually replace old endpoints
+    desc 'Add tutorial to a unit activity set'
+    params do
+      requires :tutorial, type: Hash do
+        requires :tutor_id,         type: Integer,  desc: 'Id of the tutor'
+        requires :campus_id,        type: Integer,  desc: 'Id of the campus',           allow_blank: false
+        requires :capacity,         type: Integer,  desc: 'Capacity of the tutorial',   allow_blank: false
+        requires :abbreviation,     type: String,   desc: 'The tutorials code',         allow_blank: false
+        requires :meeting_location, type: String,   desc: 'The tutorials location',     allow_blank: false
+        requires :meeting_day,      type: String,   desc: 'Day of the tutorial',        allow_blank: false
+        requires :meeting_time,     type: String,   desc: 'Time of the tutorial',       allow_blank: false
+      end
+    end
+    post '/unit_activity_sets/:unit_activity_set_id/tutorials' do
```

Comment:
I suggest just adding to the existing functionality.

The root for this should be the unit we are dealing with, then the activity set just becomes a way of grouping things.